code/game/gamemodes/miniantags/demons/shadow_demon/shadow_demon.dm

```diff
@@ -10,6 +10,17 @@
 	var/thrown_alert = FALSE
 	var/wrapping = FALSE
 
+/mob/living/simple_animal/demon/shadow/Login()
+	..()
+	to_chat(src, "<br>")
+	to_chat(src, "<span class='deadsay'><font size=3><b>You are a shadow demon.</b></font></span>")
+	to_chat(src, "<b>You are a lethal ambush predator who thrives in the darkness.</b>")
+	to_chat(src, "<b>Light is however your worst enemy and being exposed for too long will be fatal.</b>")
+	to_chat(src, "<b>Striking your victims will extinguish any light source they may carry.</b>")
+	to_chat(src, "<b>You can wrap your dead victims into a shadow cocoon which provides a shroud of darkness.</b>")
+	to_chat(src, "<b><i>You do not remember anything of your past lives, nor will you remember anything about this one after your death.</i></b>")
```

Comment:
```suggestion
	to_chat(src, "<br>")
	to_chat(src, "<span class='deadsay'><font size=3><b>You are a shadow demon!</b></font></span>")
	to_chat(src, "<b>You are a lethal ambush predator who thrives in the darkness, calling upon the shadows to heal your injured form and increase your speed.</b>")
	to_chat(src, "<b>Light is however your worst enemy and being exposed for too long will be fatal.</b>")
	to_chat(src, "<b>Striking your victims with your shadow grapple extinguishes any light sources around them. Striking items silences any light within them.</b>")
	to_chat(src, "<b>You can wrap your dead victims into a shadow cocoon which tears away any light near it.</b>")
	to_chat(src, "<b><i>You do not remember anything of your past lives, nor will you remember anything about this one after your death.</i></b>")
```
could you also make this into a list that just sends one big message vs a bunch of smaller messages